### PR TITLE
Circle ci & Netifly cd

### PR DESCRIPTION
**Pull request by lookupdaily**
Set up Circleci config:

- build, test and create a deploy preview for builds on branches other than master
- build, test and deploy master to production (upon merged PR and successful build)
_This second step needs to be validated once branch is approved and merged_

Live at [tapedit.netlify.app](https://tapedit.netlify.app)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: yarn install
+            - run: yarn test
+            - run: yarn build
+            - run: ./node_modules/.bin/netlify deploy --site $NETLIFY_SITE_ID --auth $NETLIFY_ACCESS_TOKEN --dir=build --prod --message="$CIRCLE_BRANCH@$(git log --pretty='%h %s' -n 1 $CIRCLE_SHA1)"
```
Comment (benfurber) [inline .circleci/config.yml:27]: Interesting, so circle ci is going to do the deployment for us?

Comment (lookupdaily) [reply .circleci/config.yml:27]: I believe this is the only way to incorporate testing into CD? 
If our tests fail when we merge the PR into master Netlify will go ahead and build still.

Can change if we're not worried about that?

Comment (benfurber) [reply .circleci/config.yml:27]: Was mainly just interested to see it.

**Comment by lookupdaily** [comment]
Merging as no change requests

**Pull request merged**